Documentation/gettingstarted/policy-creation.rst

```diff
@@ -12,17 +12,18 @@ Creating policies from verdicts
 
 Policy Audit Mode configures Cilium to allow all traffic while logging all
 connections that would otherwise be dropped by policy. Policy Audit Mode may be
-configured for the entire daemon using ``--policy-audit-mode=true``. When
-Policy Audit Mode is enabled, no network policy is enforced so this setting is
-**not recommended for production deployment**. Policy Audit Mode supports
-auditing network policies implemented at networks layers 3 and 4. This guide
-walks through the process of creating policies using Policy Audit Mode.
+configured for the entire daemon using ``--policy-audit-mode=true`` or for
+individual Cilium Endpoints. When Policy Audit Mode is enabled, no network
+policy is enforced so this setting is **not recommended for production
+deployment**. Policy Audit Mode supports auditing network policies implemented
+at networks layers 3 and 4. This guide walks through the process of creating
+policies using Policy Audit Mode.
 
 .. include:: gsg_requirements.rst
 .. include:: gsg_sw_demo.rst
 
-Enable Policy Audit Mode
-========================
+Enable Policy Audit Mode (entire daemon)
```

Comment:
Nit: title case, please (here and below)

Reply:
updated to title case